Add organization management documentation (#4128)
wso2/docs-is: en/identity-server/next/docs/guides/organization-management/manage-organizations.md

{% set product_name = "WSO2 Identity Server" %}
{% set admin_role_name = "admin" %}

# Set up organizations

If you are a business having a B2B (Business-to-Business) platform, you can create organizations to manage other businesses that are partnered with you.

!!! note
    Explore how you can [structure {{ product_name }} organizations]({{base_path}}/guides/organization-management/) to fit your business model.

## Prerequisites

To create organizations:

- You need to have organization management privileges in the organization (root).

## Create an organization

Follow the steps given below to create an organization.

1. First, login to your organization (root) from the {{ product_name }} Console.

> 2. On the {{ product_name }} Console, go to **Organizations** under **Organization Management** and click **Add Organization**.

Reviewer: ```suggestion
2. On the {{ product_name }} Console, go to **Organization Management** > **Organizations** and click **Add Organization**.
```

3. In the **Add Organization** dialog box that opens, enter a name and description for your organization.

    ![Create organization]({{base_path}}/assets/img/guides/organization/manage-organizations/add-organization.png){: width="500" style="display: block; margin: 0 auto; border: 0.3px solid lightgrey;"}

4. Click **Register** to add the new organization. You will now see your organization listed as below.

    ![The list of organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/organization-list.png){: width="700" style="display: block; margin: 0 auto; border: 0.3px solid lightgrey;"}

> 5. Click the **Switch** icon next to the organization to go to the organization console.

Reviewer: ```suggestion
5. Click **Switch** next to the organization to switch to the new organization's console.
```

!!! note
    As the creator of the organization, you will be assigned to the {{ admin_role_name }} role of the organization by default, which grants you the following privileges from the console.

    - Update / Delete users.
    - Create a group and assign users.
    - Assign users and groups to roles.
    - Create identity providers.
    - Update the login options and general information of the applications shared from the organization (root). Learn more about [shared applications]({{base_path}}/guides/organization-management/manage-b2b-organizations/share-applications/).
    - Configure the branding of the organization. Learn more about [branding]({{base_path}}/guides/branding/configure-ui-branding/).
    - Create organizations.

## Add organization attributes

{{ product_name }} lets you maintain additional data about the organization.

For example, you can add information about the organization's location, country, number of employees, etc.

> To add additional information about your organizations:

Reviewer: ```suggestion
To add additional information about your organization:
```

Reviewer: Shouldn't this be to add additional attributes? the remaining section explain on how to add additional attributes

1. Login to the organization (root) and go to **Organizations**.
2. Select the organization to which you wish to add additional attributes.
3. Go to the **Attributes** tab and add attributes by entering a **Name** and **Value**.
4. Click **+** to add the new organization attribute.

    ![Add organization attributes]({{base_path}}/assets/img/guides/organization/manage-organizations/add-organization-attributes.png){: width="500" style="display: block; margin: 0 auto; border: 0.3px solid lightgrey;"}

5. Click **Update** to save the configurations.

## Switch between organizations

When you have multiple [organizations]({{base_path}}/guides/organization-management/manage-organizations/) for your primary business organization, you can switch between them on the {{ product_name }} Console by selecting the required organization from the list as shown below.

![organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/organizations.png){: width="700" style="display: block; margin: 0 auto; border: 0.3px solid lightgrey;"}

> ## Create an organization for your organization

Reviewer: is the meaning of this topic ....

Create a new organization for the root organization?
